modules/govuk/manifests/apps/rummager/rabbitmq.pp

```diff
@@ -34,13 +34,14 @@
 
   # match everything on queue 2 while we test
   govuk_rabbitmq::consumer { 'rummager-v2':
```

Comment:
be aware that users can NOT be updated, hence the reason we have two users in here already (think we may be able to delete the `rummager` user as we are no longer using them, but would need to check.